.github/labeler.yml

```diff
@@ -0,0 +1,24 @@
+documentation:
+- "**/*.md"
+
+continuous_integration:
```

Comment:
not that this string "continuous_integration" corresponds to the labels for pull request found here: https://github.com/waralex/Dashboards.jl/labels

You need to make sure you use labels that actually exist or open an issue on the repo and request that the maintainer adds the PR labels you want so the labeler can work as expected.

Reply:
Sorry, I misunderstood the process. I will update soon.